tri.js

```diff
@@ -1,16 +1,56 @@
 /* Triangle Game Javascript */
 
-var circleColor = "green";
+// variables
+var circleColorOne = "green";
+var circleArrayGreen = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+var circleArrayBlack = [];
 var resetButton = document.querySelector("#reset");
```

Comment:
If you were curious, you could replace these all with the jQuery variant just by changing `document.querySelector()` 😉 
```suggestion
var resetButton = $("#reset");
```

Reply:
Vanilla JavaScript this project - next one jQuery.  Still  trying to understand the difference.  Thanks!